README.md

```diff
@@ -28,7 +28,8 @@ bug list][easybugs] and get started!
 ## Building Mixxx
 
 First, open a terminal (on Windows, use "x64 Native Tools Command Prompt for
-VS 2019"), download the mixxx source code and navigate to it:
+[VS 2019][visualstudio]"), download the mixxx
```

Comment:
Let's use `visualstudio2019`. Otherwise someone might forget to update the link behind that reference in the future, because it is out of sight here.

Reply:
Good idea